samples/aspnetcore/blazor/FlightFinder/FlightFinder.Server/Startup.cs

```diff
@@ -18,9 +18,12 @@ public class Startup
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().AddNewtonsoftJson();
-
-            services.AddResponseCompression();
+            services.AddMvc();//.AddNewtonsoftJson();
```

Comment:
Remove the commented out code?

Reply:
I'm going to put this code back in because it's still there in our templates.